pkgs/borgbackup/borg/registry.yaml

```diff
@@ -23,7 +23,7 @@ packages:
         supported_envs:
           - linux/amd64
           - darwin
-      - version_constraint: semver("<= 1.1.18")
+      - version_constraint: semver("< 1.4.0")
```

Comment:
When we update version_constraint, we should add test data to pkg.yaml to support old versions.

Reply:
Basically, we use `<=` instead of `<` for consistency.

Reply:
> Basically, we use `<=` instead of `<` for consistency.

Oh, I see.

- https://github.com/aquaproj/aqua-registry/pull/24685#discussion_r1668741697

We haven't assumed back ports, but it makes sense.

Reply:
Is there a way to express that given the current syntax/conventions? Could we use `>= 1.4.0` and leave the 1.2.x line without constraint for the time being?